src/caretogether-pwa/src/GeneratedClient.ts

```diff
@@ -2016,10 +2016,10 @@ export enum Permission {
     ViewPersonDateOfBirth = 159,
     AddEditDraftNotes = 180,
     DiscardDraftNotes = 181,
-    AddEditOwnDraftNotes = 182,
-    DiscardOwnDraftNotes = 183,
-    ApproveNotes = 184,
-    ViewAllNotes = 185,
+    ApproveNotes = 182,
+    ViewAllNotes = 183,
+    AddEditOwnDraftNotes = 184,
+    DiscardOwnDraftNotes = 185,
```

Comment:
@LarsKemmann apparently I forgot to build the app after I reordered those items, so the generated files in branch `main` are behind the actual code :cold_sweat: